### PR TITLE
use logging on front, api and renderer

### PR DESCRIPTION
**Pull request by RamonSaboya**
Effectively using logging feature.

Check info of current logging steps on: https://github.com/NeverDefineUs/vitaes/wiki/Logging-steps

### CODE REVIEW

File: vitaes-webapp/src/utils/getHostname.js [hunk 1]
```diff
@@ -1,7 +1,13 @@
-export default function getHostname() {
```
Comment (Arthurlpgc) [inline vitaes-webapp/src/utils/getHostname.js]: this is gonna blow up a bit, and use getApiHostname, since we are migrating soon

Comment (RamonSaboya) [reply vitaes-webapp/src/utils/getHostname.js]: Whe your PR goes live, I'll change it

File: vitaes-renderer/Common.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 from flask import Flask, request, abort, send_file
 from bson.objectid import ObjectId
 from CurriculumVitae import CurriculumVitae
```
Comment (Arthurlpgc) [inline vitaes-renderer/Common.py]: Rename it as it is a function

File: vitaes-renderer/Logger.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import requests
+
+def renderer_logger(email, cv_hash, step, log_data=''):
```
Comment (Arthurlpgc) [inline vitaes-renderer/Logger.py]: renames pls

File: vitaes-renderer/Logger.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    logger(email, cv_hash, 'RENDERER', step, log_data)
+
+def server_logger(email, cv_hash, step, log_data=''):
+    logger(email, cv_hash, 'SERVER', step, log_data)
+
+def logger(email, cv_hash, origin, step, log_data):
+    requests.post('http://logger:8017/', data = {
```
Comment (Arthurlpgc) [inline vitaes-renderer/Logger.py]: is that never gonna fail? that could interrupt our service if it does

File: vitaes-renderer/Renders.py [hunk 5]
```diff
@@ -248,6 +251,8 @@ def render(cv: CurriculumVitae, baseFolder: str, params={}, resources={}):
         for key in resources:
           resources[key] = text_clean(resources[key])
         cvDict["resources"] = resources
+        renderer_logger(cv.header.email, cv.cv_hash, "PRE_TEX")
```
Comment (Arthurlpgc) [inline vitaes-renderer/Renders.py]: GENERATING_TEX_FROM_CHEETAH
GENERATED_TEX_FROM_CHEETAH

File: vitaes-renderer/Logger.py (hunk 4 above)
Comment (RamonSaboya) [reply vitaes-renderer/Logger.py]: Handled

**Pull request merged**